features/managing_invoices/seeing_invoices_when_shipment_has_promotions_applied.feature

```diff
@@ -0,0 +1,24 @@
+@managing_invoices @ui
+Feature: Seeing information about an invoice with shipment has taxes and promotion applied
+    In order to see details of a specific invoice
+    As an Administrator
+    I want to be able to view basic information about an invoice
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Angel T-Shirt" priced at "$10.00"
+        And the store has "Galaxy Post" shipping method with "$20.00" fee
+        And the store allows paying with "Space money"
+        And there is a promotion "50% shipping discount"
+        And it gives "50%" discount on shipping to every order
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000666"
+        And the customer "Lucifer Morningstar" addressed it to "Seaside Fwy", "90802" "Los Angeles" in the "United States"
+        And for the billing address of "Mazikeen Lilim" in the "Pacific Coast Hwy", "90806" "Los Angeles", "United States"
+        And the customer bought 2 "Angel T-Shirt" products
+        And the customer chose "Galaxy Post" shipping method to "United States" with "Space money" payment
+        And the order "#00000666" is already paid
+        And I am logged in as an administrator
+
+    Scenario: Seeing proper invoice with a promotion applied
+        When I view the summary of the invoice for order "#00000666"
+        And it should have a shipping item "Galaxy Post" with unit price "$20.00", quantity 1, tax total "$0.00" and total "$10.00"
```

Comment:
Here you say about promotion and tax in the feature's title, promotion in scenario name... and there is no tax and no promotion checked here 🚀

Reply:
I also believe these all scenarios could be somehow unified, as there are a lot of feature files and in fact we could have one "Having promotions and taxes applied on the invoice" and check everything there